Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/Parse/Blocks/TableBlock.cs

```diff
@@ -225,7 +225,7 @@ internal static int ParseContents(string markdown, int startingPos, int maxEndin
 
                         if (c == '\r')
                         {
-                            if (pos < maxEndingPos && markdown[pos] == '\n')
+                            if (pos < maxEndingPos && (markdown[pos] == '\n' || markdown[pos + 1] == '\n'))
```

Comment:
I don't think the `markdown[pos] == '\n'` is necessary because `markdown[pos]` will always equal `\r` here